packages/design-system/src/styles/index.css

```
@@ -1,2 +1,7 @@
-@import './font.css';
+@import 'tailwindcss';
+
+@source '../components/**/*.{js,ts,jsx,tsx}';
+@layer theme;
+
+@import './font.css' layer(theme);
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)